### ISSUE TITLE
Possible Incompatiblity with Ducktales

### ISSUE THREAD

**Issue by Sowden** [ShawMerlin/NES-Slotmaster#7]
Hey guys.  I wanted to check if anyone else here is having trouble with Ducktales.  I had a No Light Blink Win in my machine before and it played every time.  I'm pretty sure that my cart is good.  Can anyone else confirm this?  Thanks guys.

**Comment by ShawMerlin** [comment]
I have not heard of Ducktales not working,  does this CART have the short
pads on it like some of the other ones that aren't working?
Also, the Blinking Light Win bypasses the CIC chip while the Slotmaster
does not, so if the light is blinking, there might be a connection or issue
with the CIC.

Thanks,
-Merlin Shaw

**Comment by Sowden** [comment]
So I just got home and tried it again.  Like I said I have a Hi-Def NES internally installed.  So games that don't seat well gives me a grey screen.  So I tried it again this morning and got the same thing.  But the red LED light was on solid red.  So I don't know what that means.  My buddy had a NES cart screw driver, and I can open it up to take a look at the board, but I'm not quite sure what I'm looking for.  If I take a picture and post it here, could you tell if it's a cart with short pins or not?

**Comment by Sowden** [comment]
Alright, so I was able open up my cart and take some photos.  Is this what you need to determine if this cart is compatible or not?  Also for the record, my copy of 1942 works with my Slotmaster, so 🤷
![Uploading IMG_20240720_092144191.jpg…]()
![Uploading IMG_20240720_092150207.jpg…]()
![Uploading IMG_20240720_092252082.jpg…]()
![Uploading IMG_20240720_092303211.jpg…]()

**Comment by Sowden** [comment]
Alright, so I was finally able to open my cart and take some photos.  This is my original cart I had as a kid, and I want to say I bought it pretty early on, but can't say for sure.  Is this what you need to determine if it's compatible or not?  Also to put on the record, my copy of 1942 worked on my Slotmaster, so 🤷

![IMG_20240720_092144191](https://github.com/user-attachments/assets/4ecfad6f-9f04-4a83-9de6-75f8379aa706)
![IMG_20240720_092150207](https://github.com/user-attachments/assets/ed78dc4e-e3c3-423c-96ee-cef431016333)
![IMG_20240720_092252082](https://github.com/user-attachments/assets/9da78c42-dcac-40c1-9b9c-b6218f435fd6)
![IMG_20240720_092303211](https://github.com/user-attachments/assets/d2768908-877a-4203-8ded-d6d4f2774f90)

**Comment by aaronperkins** [comment]
Looks like another cart with short pads. I've been documenting this issue here: https://github.com/ShawMerlin/NES-Slotmaster/issues/3

**Comment by ShawMerlin** [comment]
I have not ever seen one where it's the long pads on one side and shorter on the other.  It's worth noting that on the above picture, the short pads are not as short as the previous ones we have seen but not the full length either, it's sorta inbetween.  Thank you for documenting this!

**Comment by Sowden** [comment]
Ahh geez, that's a tad bit disappointing.  But if the rest of my games work, then that's a pretty good deal.  Thanks guys, glad I could help out

**Comment by ShawMerlin** [comment]
Im looking an option for folks that want to use their short cart games with
one of these aftermarket ports that they could swap the chips onto a brand
new PCB. It’s not 100% genuine anymore but at least the chips are still
legit and genuine and the games can be used.

**Comment by Sowden** [comment]
Oh a board swap out would be great.  Would it come from PCBway?  And would there be any soldering involved?  And thanks for working so hard for us.

**Comment by ShawMerlin** [comment]
Yes, it would be easily order-able from PCBWay and then you would just
desolder the chips and put them on the new PCB.

I hope to have the first prototype here in a few weeks and can send you
some if you shoot me an email.

There are a lot of variations of CART PCBs so they would be game specific.
I will have a cross reference chat that lists which games the board
supports.
